jbpm-task-assigning/kie-server-services-task-assigning/kie-server-services-task-assigning-core/src/main/resources/org/kie/server/services/taskassigning/solver/taskAssigningDefaultSolverConfig.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<solver>
+
+  <solutionClass>org.kie.server.services.taskassigning.core.model.TaskAssigningSolution</solutionClass>
+  <entityClass>org.kie.server.services.taskassigning.core.model.TaskOrUser</entityClass>
+  <entityClass>org.kie.server.services.taskassigning.core.model.Task</entityClass>
+
+  <daemon>true</daemon>
+
+  <scoreDirectorFactory>
+    <scoreDrl>org/kie/server/services/taskassigning/solver/taskAssigningScoreRules.drl</scoreDrl>
```

Comment:
Would you consider switching to constraint streams? The task assigning Optaplanner example already has a `ConstraintProvider` implementation that could hopefully be reused.

(It needn't be done now, just something to consider. The API is easier to work with than DRL and we would like to promote it more in the future.)

Reply:
again, my document with the configuration alternatives is in progress.
But, now, it'l be posible to configure the solver by
1) using this kind of by default option
2) start the solver from a user provided kjar, i.e. basically a container.
So at the end, we'll support the same as the current OptaPlannerKieServerExtension does.
I'll be clearer when in the configuration document, but, to summarize
as soon the user provided kjar is properly configured, the solver can be started from that container. Finally if the SolverFactory.createFromKieContainerXmlResource(XXX) supports configuring the Solver based on ConstraintStreams, all will work good. If it doesn't, then we have a problem both in the OptaplannerKieServerExtension an in the PlanningKieServerExtension, but for sure not a big issue.